x/incentive/keeper/keeper_test.go

```diff
@@ -144,7 +144,7 @@ func (suite *KeeperTestSuite) TestGetSetSwapRewardIndexes() {
 			},
 		},
 		{
-			name:     "indexes with empty pool name can be written and read",
+			name:     "indexes with empty pool name panics",
```

Comment:
The pool name is used as a keeper store key in `Set/GetSwapRewardIndexes`, with the cosmos changes that don't allow for empty keys it panics now. We could add some prefix or some non-empty value if we need to support an empty pool name, but briefly looking through where the keeper method is used it seems like it should be non-empty since it uses things like reward period collateralType

Reply:
Pool names only come from params and they should always be non-empty, there's a check in params validation.